### PR TITLE
Add HPE 5510 48G POE and non-POE

### PR DESCRIPTION
**Pull request by ianh14721**
Add two HPE switches:
- 5510 48G 4SFP+ HI (JH146A)
- 5510 48G PoE+ 4SFP+ HI (JH148A)

### CODE REVIEW

File: device-types/HPE/5510-48G-4SFPP-HI.yaml [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+console-ports:
+  - name: console
+    type: rj-45
+power-ports:
+  - name: 1
+    type: iec-60320-c14
+    maximum_draw: 150
+  - name: 2
+    type: iec-60320-c14
+    maximum_draw: 150
```
Comment (DanSheps) [inline device-types/HPE/5510-48G-4SFPP-HI.yaml:17]: PSU's are modular and should be modelled as modules

File: device-types/HPE/5510-48G-POEP-4SFPP-HI.yaml [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+console-ports:
+  - name: console
+    type: rj-45
+power-ports:
+  - name: 1
+    type: iec-60320-c16
+    maximum_draw: 670
+  - name: 2
+    type: iec-60320-c16
+    maximum_draw: 670
```
Comment (DanSheps) [inline device-types/HPE/5510-48G-POEP-4SFPP-HI.yaml:17]: PSU's are modular and should be modelled as such

**Review by DanSheps — CHANGES_REQUESTED** [review]

File: device-types/HPE/5510-48G-4SFPP-HI.yaml (hunk 1 above)
Comment (ianh14721) [reply device-types/HPE/5510-48G-4SFPP-HI.yaml:17]: Updated.

Comment (DanSheps) [reply device-types/HPE/5510-48G-4SFPP-HI.yaml:17]: This isn't quite what I meant.

You would convert these two to 2 module bays, label them "PSU 1"/"PSU2" with positions of 1 and 2.  Then whoever uses this DT would insert the associated module.

**Review by DanSheps — CHANGES_REQUESTED** [review]

File: device-types/HPE/5510-48G-4SFPP-HI.yaml (hunk 1 above)
Comment (ianh14721) [reply device-types/HPE/5510-48G-4SFPP-HI.yaml]: Ah got it, I think. New commits should show both the device-type with module bays, and the module-type for the power supplies. I used device-types/Cisco/UCS-FI-6248UP.yaml and module-types/Cisco/UCS-PSU-6248UP-AC.yaml as examples - are they right?

Thanks :)

**Comment by ianh14721** [comment]
@DanSheps is there anything further required for this?

**Pull request merged**